SpellChecker.java

```diff
@@ -8,28 +8,65 @@ public static void main(String[] args) {
 		String[] dictionary = readDictionary("dictionary.txt");
 		String correction = spellChecker(word, threshold, dictionary);
 		System.out.println(correction);
+		//System.out.println(levenshtein(word,"control"));
+
 	}
 
-	public static String tail(String str) {
-		// Your code goes here
+	public static String tail(String str) 
+	{
+		if(str.length()==1)
+		{
+			return "";
+		}
+		 return str.substring(1);
 	}
 
-	public static int levenshtein(String word1, String word2) {
-		// Your code goes here
+	public static int levenshtein(String word1, String word2) 
+	{
+		if(word1.length()==0)
+			return word2.length();
+		if(word2.length()==0)
+			return word1.length();
+		word1 = word1.toLowerCase();
+		word2 = word2.toLowerCase();
```

Comment:
Generally it's not common to override method params value, better to store manipulated data within new variables.